demo-shell/src/app/components/cloud/community/community-processes-cloud.component.scss

```diff
@@ -0,0 +1,21 @@
+:host {
```

Comment:
can we remove the `:host` and use `ViewEncapsulation.none` instead, as this is the remains of some old code. Better scoping the styles to the component style

Reply:
Sure I will change it